graphql-dgs-codegen-gradle/src/main/kotlin/com/netflix/graphql/dgs/codegen/gradle/ClientUtilsConventions.kt

```diff
@@ -38,10 +40,23 @@ object ClientUtilsConventions {
         optionalCodeClientDependencyScope: Optional<String> = Optional.empty()
     ) {
         clientCoreArtifact(optionalCodeUtilsVersion).ifPresent { dependencyString ->
+            val dependencyLockString = dependencyString.split(":").take(2).joinToString(":")
+
             val dependencyConfiguration = optionalCodeClientDependencyScope.orElse(GRADLE_CLASSPATH_CONFIGURATION)
             val configurationDependencies = project.configurations.getByName(dependencyConfiguration).dependencies
             configurationDependencies.add(project.dependencies.create(dependencyString))
             logger.info("DGS CodeGen added [{}] to the {} dependencies.", dependencyString, dependencyConfiguration)
+
+            project.plugins.withId(CLIENT_UTILS_NEBULA_LOCK_ID) {
+                val extension = project.extensions.getByType(DependencyLockExtension::class.java)
+                if (extension != null) {
+                    logger.info("DGS CodeGen added [{}] to the skippedDependencies.", dependencyLockString, dependencyConfiguration)
+                    extension.skippedDependencies.add(dependencyLockString)
+                }
+
+                logger.info("DGS CodeGen added [{}] to the ignoredDependencies.", dependencyLockString, dependencyConfiguration)
+                project.dependencyLocking.ignoredDependencies.add(dependencyLockString)
```

Comment:
I was wrong! This configuration is actually for native gradle locking so it is [supporting](https://docs.gradle.org/current/userguide/dependency_locking.html#ignoring_dependencies) ignoring selected dependencies now. It means this line should be actually outside of `withId`.

Reply:
Good to know, that's better! I will fix

Reply:
Do you have any minimum requirement for Gradle version? That might double check that it is not changing. I'm not sure when ignored dependencies in gradle locking got added.

Reply:
Looks like Gradle 6.7 will be the first version with this ignore lock feature.

Reply:
@srinivasankavitha to weigh in. I cannot locate any minimum Gradle version requirement in Codegen, and if we're not compatible with that requirement, I can add some null checking.

Reply:
Looks like we started supporting around 7.0.2. We don't enforce any minimum version, but given that 6.x is fairly outdated by now, should beok?